File: 프로그래머스/0/120817. 배열의 평균값/배열의 평균값.js
function solution1(numbers) {
    let sum = 0;
    for (let i = 0; i < numbers.length ; i++){
        sum += numbers[i]
    }
    return sum / numbers.length
}

function solution2(numbers) {
    let sum = 0;
    numbers.forEach(num => {
        sum += num
    })
    return sum / numbers.length
}

function solution(numbers) {
    let sum = numbers.reduce((a,b) => a+b, 0)
    return sum / numbers.length
}

File: 프로그래머스/0/120817. 배열의 평균값/배열의 평균값.cpp
#include <string>
#include <vector>

using namespace std;

double solution(vector<int> numbers) {
    double sumv = 0;
    for(int i = 0; i < numbers.size(); i ++){
        sumv += numbers[i];
    }
    return sumv / numbers.size();
}